rcl/src/rcl/arguments_impl.h

```diff
@@ -50,8 +51,8 @@ typedef struct rcl_arguments_impl_t
   /// Length of remap_rules.
   int num_remap_rules;
 
-  /// Default log level (represented by `RCUTILS_LOG_SEVERITY` enum) or -1 if not specified.
-  int log_level;
+  /// Log levels parsed from arguments.
+  rcl_log_levels_t * log_levels;
```

Comment:
as commented elsewhere, this could be directly:
```suggestion
  rcl_log_levels_t log_levels;
```

Reply:
Updated